public/app/core/utils/datemath.ts

```diff
@@ -1,9 +1,10 @@
+// @ts-ignore
 import _ from 'lodash';
 import moment from 'moment';
 
 const units = ['y', 'M', 'w', 'd', 'h', 'm', 's'];
 
-export function parse(text, roundUp?, timezone?) {
+export function parse(text: any, roundUp?: any, timezone?: any) {
```

Comment:
Can we be more precise here when it comes to types?

Reply:
I could certainly give it a try :)

Reply:
Created new issue for this #15410